config/dependency_decisions.yml

```
@@ -139,3 +139,9 @@
     :why: Compatible with Apache-2.0 license. See https://opensource.org/license/artistic-2-0
     :versions: []
     :when: 2025-05-08 05:24:20.998252000 Z
+- - :approve
+  - "@mastra/core"
+  - :who: Liam @FunamaYukina
+    :why: The license is Elastic-2.0. License Finder failed to detect the license.
```

@FunamaYukina

LicenseFinder correctly detects the license for `@mastra/core`.

```console
$ license_finder
LicenseFinder::PNPM: is active for '/Users/masutaka/src/github.com/liam-hq/liam'

Dependencies that need approval:
@mastra/core, 0.9.3, Elastic-2.0
```

I think you should just approve Elastic-2.0 itself.

```suggestion
- - :permit
  - Elastic-2.0
  - :who: Liam @FunamaYukina
    :why: Compatible with Apache-2.0 license. See https://www.elastic.co/jp/licensing/elastic-license
```

---

Elastic 2.0. would be a big decision to allow, so should we revert back to the original?
```
- - :approve
  - "@mastra/core"
  - :who: Liam @FunamaYukina
    :why: The license is Elastic-2.0. License Finder failed to detect the license.
    ```

---

@FunamaYukina

That's right.
I think you should change the why.

Example:

```yaml
:why: The license is Elastic-2.0. This is not an OSS license, so to be on the safe side, we'll approve it on a package-by-package basis.
```

---

I think that's a good idea too. 
If we are going to use Elastic 2.0 libraries other than mastra in the future, we will need to check.

---

I have rewritten it.👍
https://github.com/liam-hq/liam/pull/1664/commits/d3f712309d025139ed0852f3f83ba62c53e27967